ll-list-control.html

```diff
@@ -38,67 +38,79 @@
+        <div class="col-sm-4">
+          <h2>[[controlTitle]]</h2>
+        </div>
+        <div class="col-sm-8">
+          <div class="pull-right">
+            <div id="addItemButton" class="btn btn-primary pull-right" on-tap="_fireAssociateNewSelected">[[associateText]]</div>
+            <div id="addNewButton" class="btn btn-default pull-right" on-tap="_fireCreateNewSelected" style="margin-right:15px;">Create New</div>
+            <div id="removeItemButton" class="btn btn-destructive pull-right" on-tap="_fireRemoveSelected" style="margin-right:15px;display:none;">Remove Selected</div>
+          </div>
+        </div>
+      </div>
+      <hr>
+      <template id="item-list" is="dom-repeat" items="[[items]]" >
+        <div>
+          <div style="margin-bottom:10px;" class="row">
+            <div class="col-sm-1 col-xs-2 text-center">
+              <input type="radio" name=[[radioButtonGroupName]] id="[[item.id]]" on-tap="_prepareForRemoval">
+            </div>
+            <div class="col-sm-3 col-xs-10">[[item.name]]</div>
+          </div>
+        </div>
+      </template>
+      <p id="listEmpty" style="display: block;">No properties to display <a href="#" id="addItemLink">Add one now &gt;</a></p>
+      <hr>
+    </div>
   </template>
 
 </dom-module>
 
 <script>
+  var selectedItemId = '';
 
   Polymer({
 
     is: 'll-list-control',
 
     properties: {
-
       /**
-       * `fancy` indicates that the element should don a monocle and tophat,
-       * while checking its pocket watch.
+       * The Title of the control
        */
-      fancy: Boolean,
-
+      controlTitle: {
+        type: String,
+        default: ''
+      },
+      /**
+       * The text to display on the associate existing button
+       */
+      associateText: {
+        type: String,
+        default: 'Associate New'
+      },
       /**
-       * Describes the author of the element, but is really just an excuse to
-       * show off JSDoc annotations.
+       * The items to display
        *
-       * @type {{name: string, image: string}}
+       * @type {[{_id: string, title: string, description: string, isDefault: boolean, url: string}]}
        */
-      author: {
-        type: Object,
-        // Use `value` to provide a default value for a property, by setting it
-        // on your element's prototype.
-        //
-        // If you provide a function, as we do here, Polymer will call that
-        // _per element instance_.
-        //
-        // We do that to ensure that each element gets its own copy of the
-        // value, rather than having it shared across all instances (via the
-        // prototype).
-        value: function() {
-          return {
-            name:  'Dimitri Glazkov',
-            image: 'http://addyosmani.com/blog/wp-content/uploads/2013/04/unicorn.jpg',
-          };
-        }
+      items: {
+        type: Array,
+        notify: true,
+        default: []
       },
-
+      radioButtonGroupName: {
```

Comment:
Needs some comments here. The comments are used to create the docs when you run 

```polyserve```

Looking at the generated docs, it doesn't seem to be picking up any of your docs.